filebeat/input/file/state.go

```diff
@@ -33,6 +34,20 @@ func NewState(fileInfo os.FileInfo, path string, t string) State {
 	}
 }
 
+// String returns a unique id for the state as a string
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported method State.ID should be of the form "ID ..."